src/styles/common.scss

```diff
@@ -178,6 +178,7 @@
 
 .linkCard {
     position: relative;
+    aspect-ratio: 0.75;
```

Comment:
Why?

Reply:
to remove `aspect-ratio: 2.26;` for entire row and to not set` aspect-ratio: 0.75;` in media queries where cards aligned vertically.

Reply:
IMO it doesn't make sense to hardcode aspect ratio into a link card.